consensus/clique/src/test/java/tech/pegasys/pantheon/consensus/clique/blockcreation/CliqueBlockCreatorTest.java

```diff
@@ -109,7 +110,7 @@ public void proposerAddressCanBeExtractFromAConstructedBlock() {
         new CliqueBlockCreator(
             coinbase,
             parent -> extraData.encode(),
-            new PendingTransactions(5),
+            new PendingTransactions(5, Clock.systemUTC()),
```

Comment:
Could we use the fixed clock in tests please?  Real-time in tests comes back to bite you.  Might need a util method somewhere to make it easy since `Clock.fixed` requires specifying a time and we generally don't care what time is used.

Reply:
Added a TestClock with a fixed() static method.